bundle/config/mutator/process_target_mode_test.go

```diff
@@ -59,6 +61,10 @@ func mockBundle(mode config.Mode) *bundle.Bundle {
 				},
 			},
 		},
+		// Use AWS implementation for testing.
+		Tagging: tags.ForCloud(&sdkconfig.Config{
+			Host: "https://company.cloud.databricks.com",
+		}),
```

Comment:
Could you also add a few cloud-specific tests? Or at least something that show that cloud-specific normalization works for funky tag values with AWS?

Reply:
Added tests to cover this. Please TAL.